### PR TITLE
Update API structure

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

Addressing https://github.com/soramitsu/iroha2-block-explorer-backend/pull/25#discussion_r909542701

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+| Domain Id           | `wonderland`       | `wonderland`        |
+| Account Id          | `alice@wonderland` | `alice@wonderland`  |
+| Asset Definition Id | `rose#wonderland`  | `rose%23wonderland` |
+
+## Pagination Query Params
+
+|        Param         | Default |   Description   |
+| -------------------- | :-----: | --------------- |
+| `page=<number>`      |    1    | Page number     |
+| `page_size=<number>` |   15    | Page size limit |
```
Comment (0x009922) [inline api.md:212]:
```suggestion
| Param       | Type     | Default | Description     |
| ----------- | -------- | :-----: | --------------- |
| `page`      | `number` |    1    | Page number     |
| `page_size` | `number` |   15    | Page size limit |

```

File: api.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  | Param |   Type   |     Description      |
+  | :---: | :------: | -------------------- |
+  | `id`  | `string` | The id of the domain |
+
+- **Response**: `Domain` or `404`
+
+### Peers
+
+- [`/peer/peers`](#get-apiv1peerpeers)
+- [`/peer/status`](#get-apiv1peerstatus)
```
Comment (0x009922) [inline api.md:180]: I'd name this section "Peer data" (the data of the concrete peer that the explorer is connected to) instead of "Peers"

**Review by 0x009922 — APPROVED** [review]

File: api.md (hunk 2 above)
Comment (outoftardis) [reply api.md:180]: well it's "data" for all sections, not only this one, isn't it? this is only to indicate which entity the queries are related to. I think if we add "data" here, we'd need to do it to all other sections to be consistent

Comment (0x009922) [reply api.md:180]: I agree about "data", but I think the main point is "Peer" instead of "Peers".

File: api.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+- **Params**:
+
+  | Param |   Type   |     Description      |
+  | :---: | :------: | -------------------- |
+  | `id`  | `string` | The id of the domain |
+
+- **Response**: `Domain` or `404`
+
+### Peers
```
Comment (outoftardis) [inline api.md:177]:
```suggestion
### Peer
```

File: api.md (hunk 2 above)
Comment (outoftardis) [reply api.md]: changed to "Peer", ok?

Comment (0x009922) [reply api.md]: Ok!

**Pull request merged**